File: Leetcode/two_pointers/674_Longest Continuous Increasing Subsequence/Solution.cpp
class Solution {
public:
    int findLengthOfLCIS(vector<int>& nums) {
        if(nums.size()==0) return 0;
        int res = 1, slow = -1, fast = 0;
        for(fast = 0; fast < nums.size()-1; ++fast){
            if(nums[fast]<nums[fast+1]){
                continue;
            }
            res = max(fast-slow, res);
            slow = fast;
        }
        return max(res, fast-slow);
    }
};

File: Leetcode/two_pointers/674_Longest Continuous Increasing Subsequence/Solution.py
class Solution(object):
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums)==0:
            return 0
        res = 1
        slow = -1
        fast = 0
        while fast < len(nums)-1:
            if(nums[fast]>=nums[fast+1]):
                res = max(fast-slow, res)
                slow = fast
            fast += 1
        return max(res, fast-slow)
